### ISSUE TITLE
Guilds - Characters A-Z

### ISSUE THREAD

**Issue by oOChrisOo** [Wynncraft/Issues#8403]
Whenever I try to create my Guild, (I'm Level 56 with 6 LE), I type in an English based name with only letters A-Z, and I continue to receive the message requiring me to use only English characters. The input sign continues to pop up until I switch servers of log off as it never accepts my name input.

**Comment by thecoolkogut4** [comment]
If you don't mind me asking, what exactly are you trying to name the guild?

**Comment by oOChrisOo** [comment]
Names like: Boundless, Primus, The Rising Tide, casual names that are definitely A-Z

**Comment by thecoolkogut4** [comment]
Does #7162 help out?

**Comment by oOChrisOo** [comment]
Not really, Primus should be working fine but it doesn't. It's short and concise.

**Comment by thecoolkogut4** [comment]
If you have any friends that could help you out, maybe get them to try and make the guild then transfer it over to you?

**Comment by LordHelmchen98** [comment]
We're you able to solve this issue?
